crates/bifrost/src/bifrost.rs

```diff
@@ -681,7 +690,14 @@ mod tests {
             appender.append("").await?;
         }
 
-        bifrost_admin.trim(LOG_ID, Lsn::from(5)).await?;
+        assert_eq!(bifrost_admin.trim(LOG_ID, Lsn::INVALID).await?, None); // no-op
+        assert_eq!(Lsn::INVALID, bifrost.get_trim_point(LOG_ID).await?);
+
+        assert_eq!(
+            bifrost_admin.trim(LOG_ID, Lsn::from(5)).await?,
+            Some(Lsn::from(5))
+        );
+        assert_eq!(bifrost_admin.trim(LOG_ID, Lsn::from(5)).await?, None); // no-op
```

Comment:
Not sure whether the second `trim` call returning `None` is very intuitive and useful. To me it would make slightly more sense if the second call would also return `Some(Lsn::from(5))`.